devops/llvm-anybuild.yml

```diff
@@ -7,9 +7,26 @@ parameters:
   - name: BuildDebug
     type: boolean
     default: true
+  # extra command-line args to pass to AnyBuild
   - name: ExtraAnyBuildArgs
     type: string
-    default: ' '
+    default: '--Epoch 1'
+  # LLVM commit hash to build
+  - name: GOOD_HASH
+    type: string
+    default: '73811d3'
+  # args to pass to cmake when building
+  - name: CmakeBuildToolArgs
+    type: string
+    default: 'install'
+  # number of parallel cmake jobs
+  - name: CmakeJobs
+    type: number
+    default: 32
```

Comment:
Just a note, but if you're using Ninja you don't need this option. Ninja works on Linux, Mac and Windows.

Alternatively, you could also get the number of cores with a script and use that instead.

Reply:
Here the idea is to use many more jobs than the number of cores available, because all those compilation processes will be executed remotely.  So this number really depends on how big the AnyBuild cluster is.

Reply:
True, I forgot about that. Looks good, thanks!